app/controllers/lessons_controller.rb

```diff
@@ -1,6 +1,7 @@
 class LessonsController < ApplicationController
   def show
     @lesson = decorated_lesson
+    @content = lesson.content
```

Comment:
Nit: The view code needs updated to use this new instance variable.

_But_ I'd  also be kind of tempted to go with `lesson.content.body.html_safe` in the view instead of introducing a new instance variable for content. I think removing that layer of indirection will make it clearer where content is coming from in the view without having to open the controller.

We could delegate body in the lesson model to get past the law of demeter violation this change would add.

Reply:
I originally did have it with the new ivar but it got swallowed by my last rebase :laughing: good catch :muscle: 

Demeter was the original motivation for the exta ivar but I agree that's it's probably not worth it. I'm a fan of the delgation approach in general and I think it works well here :ok_hand: